contracts/utils/Strings.sol

```diff
@@ -438,4 +447,42 @@ library Strings {
             value := mload(add(buffer, add(0x20, offset)))
         }
     }
+    
+    function _escapeJsonString(string memory input) private pure returns (string memory) {
+    bytes memory buffer = bytes(input);
+    bytes memory output = new bytes(buffer.length * 2); // Allocate max possible space
```

Comment:
I'm worried this allocated a lot of memory that is not freed. This is quite expensive. Alternatives are:
- count number of "extra chars" in a loop, and allocate the right size directly
- resize the output buffer without a copy, and move the free memory ptr, using assembly